webskins/_default_/tmpl/BaseHeader.tmpl

```diff
@@ -6,7 +6,7 @@
 <? AddRow JSLoop HREF=/pub/selectize-standalone-0.12.1.min.js ?>
 <? AddRow CSSLoop HREF=/pub/jquery-ui-sortable.1.11.4.min.css ?>
 <? AddRow CSSLoop HREF=/pub/selectize-0.12.1.css ?>
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<html lang="en" dir="ltr">
```

Comment:
Why did you make it ltr? Do you think for Arabic/Hebrew support something needs to be changed in the markup?